lib/pure/asyncnet.nim

```diff
@@ -456,7 +456,8 @@ proc bindAddr*(socket: AsyncSocket, port = Port(0), address = "") {.
     of AF_INET6: realaddr = "::"
     of AF_INET:  realaddr = "0.0.0.0"
     else:
-      raiseOSError("Unknown socket address family and no address specified to bindAddr")
+      raise newException(OSError,
+        "Unknown socket address family and no address specified to bindAddr")
```

Comment:
What's this? 'raiseOSError' produces smaller code. And smaller code is better for cold code paths.